ci/azure-pipelines-nightly.yml

```diff
@@ -124,6 +124,28 @@ jobs:
     displayName: "Run the tests"
   - template: ../ci/publish-bazel-test-results.yml
 
+- job: cryptotest_cw310
+  displayName: "Cryptotest Framework CW310 Tests"
+  timeoutInMinutes: 300
```

Comment:
This is an estimate of what the timeout will be once all of the crypto tests are included. The algorithms that are currently merged at the time of this writing (ECDSA, SHA-2, SHA-3, SHAKE) should complete in < 90 minutes.

Reply:
Edit: HMAC, DRBG, and ECDH are now included in master as well (see #21976)